composer.json

```diff
@@ -5,8 +5,8 @@
   "require" : {
     "php" : ">=5.3.3",
     "orkestra/common" : "~1.0",
-    "guzzle/guzzle" : "~3.7",
-    "symfony/options-resolver" : "~2.3"
+    "guzzlehttp/guzzle" : "~3.7",
+    "symfony/options-resolver" : "~3.0"
```

Comment:
Let's change this to `"symfony/symfony": "~2.3|~3.0",`

Otherwise, `Symfony\Component\OptionsResolver\OptionsResolver` has existed since 2.1-- I think we are safe with just a point release (keeping master on 1.1.x)